infrastructure/openapi.yaml

```diff
@@ -28,15 +21,35 @@ paths:
             type: string
       responses:
         200:
-          description: good response
+          description: Successfully retrieved character data
+      x-amazon-apigateway-integration:
+        httpMethod: POST
+        payloadFormatVersion: "1.0"
+        type: "aws_proxy"
+        uri: "arn:aws:apigateway:${aws_region}:lambda:path/2015-03-31/functions/${warmane_analytics_api_v2_main_function_arn}/invocations"
+    options:
```

Comment:
Is it possible to use the `*` route for this? Assuming that we have a whole bunch of endpoints, it would be unfortunate to have to repeat this all of the time.